partner_address_street3/view/partner_view.xml

```diff
@@ -7,18 +7,27 @@
       <field name="inherit_id" ref="base.view_partner_form"/>
       <field name="arch" type="xml">
         <xpath
-            expr="//form[@string='Partners']/sheet/group/group/div/field[@name='street2']"
+            expr="/form/sheet/group/group/div/field[@name='street2']"
```

Comment:
What would be the benefit of this? If someone completely rewrites the view and wants to use this addon, he will have to integrate street3 himeself.

There are 3 fields @name=street2  in the view, so I need to tell which precisely.